spk/sonarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = sonarr
+SPK_VERS = $(shell date +%Y%m%d)
```

Comment:
Wouldn't it be better to freeze this ?

Reply:
If I recall from previous comments from @hgy59, the dates are used with the `*arr` and other apps which have internal update mechanisms. In these apps, the internal version of the app can and does change, whereas fixed versions are used for apps where the package is tied to that version (like with `SABnzbd`).

Reply:
I suppose I could set the date to the source file (which would be 20230306) but then we would need to maintain data on the source file both in `cross/sonarr` and `spk/sonarr`. This seems like a maintenance nightmare to me.

@hgy59, any thoughts on how we should proceed here? Are there any variables we can use to pull the date from the downloaded source file?

Reply:
> Nope, this won't work. There is then no way we can reproduce this spk afterwards. This date should be statically set and probably match the creation date of version 4.0.0.394 or corresponding to the md5sum of the commit associated to it.

@th0ma7 sorry to disagree. Packages that use app internal updaters have been published this way. The package version is taken from the date at build time (plus the usual REV).
- we do not use the real version number of the included app, since it is not applicable when the already installed package has updated to a newer version
- we need to create updates for such packages only, when the (generic) installer code or a dependen library in the package needs an update (for example: when new archs are available, or the dependen python changes from 310 to 311, ...)
- we could use the "release date" of the bundled app, but this would not change the fact that dependent modules where updated inbetween (for example python 3.10 had a minor version update)

IMHO we should keep this convention and use `SPK_VERS = $(shell date +%Y%m%d)` for packages with an internal updater.
cc @publicarray, @ymartin59 


BTW it is a good practice to add the version of the bundled app to the changelog of such packages. If there is no explicit version, this could be a git sha plus commit date.